src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -69,7 +76,10 @@ const getAccountNumberSchema = (implicitParams: {
   },
   country: { name: 'country', value: implicitParams.country },
   fiatAccountType: { name: 'fiatAccountType', value: FiatAccountType.BankAccount },
-  accountName: { name: 'accountName', value: 'n/a' },
+  accountName: {
+    name: 'accountName',
+    computeValue: ({ accountNumber }) => getObfuscatedAccountNumber(accountNumber ?? ''),
```

Comment:
https://github.com/valora-inc/wallet/pull/2667/files#r912256759 did we want to do this here or a separate PR?

Reply:
checking on slack what Jacob thinks since it depends also on his pr